rc/tools/git.kak

```diff
@@ -39,6 +39,22 @@ hook -group git-show-branch-highlight global WinSetOption filetype=git-show-bran
     hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/git-show-branch}
 }
 
+hook -group git-format-patch global WinSetOption filetype=git-format-patch %{
+    require-module diff
```

Comment:
Why require the `diff` module if you don't use it?

Reply:
I didn't compile without it.